modules/Bio/EnsEMBL/Registry.pm

```diff
@@ -1818,6 +1820,12 @@ sub load_registry_from_db {
         $taxonomy_db      = $db;
     } elsif ( $db =~ /^ensembl_metadata$/ ) {
         $ensembl_metadata_db      = $db;
+    }
+    elsif ( $db =~ m{ \A ensembl_metadata_(\d+) \z }msx ) {
+      if ( $1 eq $software_version ) {
+        $ensembl_metadata_db_versioned = $db;
+        $ensembl_metadata_version      = $1;
```

Comment:
`$ensembl_metadata_version` is only used elsewhere to indicate that a versioned metadata db has been found. Were you intending to use it elsewhere, or is this line to help clarify the regex? Otherwise, line 2360 and line 2371 could be updated to check for 
`( defined $ensembl_metadata_db_versioned ) ` with the same functionality.